packages/ipython.vm/ipython.vm.nuspec

```
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>ipython.vm</id>
-    <version>8.27.0.20241001</version>
+    <version>8.27.0</version>
```

you have decreased the version (by removing the date) causing that our CI is testing the old package version without the version, which now fails as it is required. Ensure you increase the version:
```suggestion
    <version>8.27.0.20250122</version>
```

---

@Ana06 changes have been addressed